vendor/modules.txt

```diff
@@ -538,7 +550,7 @@ golang.org/x/oauth2/jwt
 # golang.org/x/sync v0.0.0-20200317015054-43a5402ce75a
 ## explicit
 golang.org/x/sync/errgroup
-# golang.org/x/sys v0.0.0-20200523222454-059865788121
+# golang.org/x/sys v0.0.0-20200810151505-1b9f1253b3ed
```

Comment:
should we use a version of the new modules that work with our existing version of `golang.org/x/sys` to minimize changes?

Reply:
I'll try and see if there is a version that supports the same version of `golang.org/x/sys` 👍🏼